src/Illuminate/Database/Eloquent/FactoryBuilder.php

```diff
@@ -45,16 +45,16 @@ class FactoryBuilder
     /**
      * Create an new builder instance.
      *
-     * @param  string  $class
-     * @param  string  $name
-     * @param  array  $definitions
-     * @return void
+     * @param  string $class
+     * @param  string $name
+     * @param  array $definitions
+     * @param \Faker\Generator $faker
      */
-    public function __construct($class, $name, array $definitions)
+    public function __construct($class, $name, array $definitions, \Faker\Generator $faker = null)
```

Comment:
The second parameter is expecting a Faker\Generator instance, but a string is given at DatabaseServiceProvider:59